src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const numbers = [];
+
+  return function InputOperator(number) {
+    if (number) {
+      numbers.push(number);
+
+      return InputOperator;
+    } else {
+      const accomulatedNumbers = [...numbers];
+
+      numbers.length = 0;
+
+      return accomulatedNumbers.reduce((acc, curV) => acc + curV, 0);
```

Comment:
instead of using reduce, you can store the sum of numbers as a number, not as an array (array takes much more machine memory than number)